chacractl/api/binaries.py

```diff
@@ -136,6 +136,8 @@ def main(self):
                 self.post(url, filepath)
 
         elif self.parser.has('delete'):
+            if self.parser.get('delete') is None:
```

Comment:
oh wait, I think I messed up here. `parser.has` is meant to check a flag but here we want a url so `self.parser.get` is the correct thing on line 138. Then you don't need to check again if it is `None`